gcloud/_helpers.py

```diff
@@ -48,6 +51,7 @@
     (?P<nanos>\d{1,9})                       # nanoseconds, maybe truncated
     Z                                        # Zulu
 """, re.VERBOSE)
+DEFAULT_CONFIGURATION_PATH = '~/.config/gcloud/configurations/config_default'
```

Comment:
Actually as I think about it. I'm not 100% sure where `gcloud auth login` would put the file in windows.